Dockerfile

```diff
@@ -17,3 +17,7 @@ RUN apt-get update \
     && apt-get autoremove -y \
     && rm -rf /var/lib/apt/lists/* \
     && rm -rf /root/.cache/pip
+
+COPY ./demo .
+RUN pip install -r requirements.txt
+CMD [ "streamlit", "run", "app.py" ]
```

Comment:
This Dockerfile is meant to use the python library, we might add a dockerfile specific for the demo but for now, let's not modify this file :pray:

Reply:
Yep, sorry about this. Will do